src/lib.rs

```diff
@@ -331,7 +331,10 @@ impl<Fs: FileSystem> ResolverGeneric<Fs> {
                 if !starts_with_node {
                     specifier = format!("node:{specifier}");
                 }
-                return Err(ResolveError::Builtin(specifier));
+                return Err(ResolveError::Builtin {
+                    resolved: specifier,
+                    prefixed_with_node_colon: !starts_with_node,
```

Comment:
Shouldn't this be `prefixed_with_node_colon: starts_with_node` if it means "it had 'node:' prefx"?

Reply:
`prefixed_with_node_colon` means the `resolved` string is `prefixed` with `node:` by `oxc_resolver`, user could get the original `resolved` string with `prefixed_with_node_colon` flag, it did not means the original `specifier` is starts with `node:`, feel free use a better naming

Reply:
I just can't think of a better name.

Reply:
> prefixed_with_node_colon means the resolved string is prefixed with node:

The resolved specifier is always prefixed with `node:`.

Does it make sense to just change it to `prefixed_with_node_colon: starts_with_node`?

Reply:
I would advise thinking in a more generic way; Node.js isn't the only runtime that has built-in modules. For example, Cloudflare has their [`cloudflare:`](https://developers.cloudflare.com/workers/runtime-apis/tcp-sockets/) modules. 

You could call it `is_runtime_module`, maybe. Or just `is_runtime`. And explain that those are modules that are runtime specific, e.g. `node:`, `cloudflare:`, `bun:`, etc.